dsgrid/config/dimension_mapping_base.py

```diff
@@ -168,6 +168,16 @@ class DimensionMappingBaseModel(DSGBaseModel):
         title="to_dimension",
         description="To dimension",
     )
+    from_fraction_tolerance: Optional[float] = Field(
+        title="from_fraction_tolerance",
+        description="Tolerance value to apply to the from_fraction column",
```

Comment:
"Tolerance to apply when checking from_fraction column sums"? (And similar edit for to_fraction_tolerance description?)